### ISSUE TITLE
Add logging and save to output

### ISSUE THREAD

**Issue by lmperry** [scitran-apps/structural-analysis#3]

**Comment by cgc** [comment]
i don't think my gear should handle this as it's a missing feature from flywheel. flywheel should show me the exit code of my gear and it should show me the stdout and stderr of my gear execution.

Carlos Correa
Scientific Data Curator
________________________________

**Comment by lmperry** [comment]
I agree with you, I have had that thought as well. However, please consider that these gears don't _have_ to be run within Flywheel at all and can be run locally. While they are built to the Flywheel spec that's not the only use case.

**Comment by cgc** [comment]
I do agree that we should consider both the production, development, and other contexts for flywheel gears. Because stdout and stderr and unix process exit codes are accessible to anyone directly running the `docker run` commands, I think we should defer to the runtime/platform (docker or flywheel in our cases) to support any logging, particularly if the logging isn't entirely verbose.
